tmpfiles.sh

```diff
@@ -29,6 +29,20 @@ checkprefix() {
 	return 1
 }
 
+owned_by_root() {
+	local path="$1"
+	if [ -z "${path}" ] ; then
+		echo "Missing parameter" >&2
+		return 404
+	fi
+	FOUND=$(find "${path}" -maxdepth 0 -uid 0 -gid 0)
+	if [ -z $FOUND ] ; then
```

Comment:
afaict, `FOUND` is only used here, so there's no need to create it at all.  this line also lacks quoting.

`if [ -z "$(find "${path}" -maxdepth 0 -uid 0 -gid 0)" ] ; then`

Reply:
Changed in commit 4f4576a6781fb64a6c798d131f8309802c2f9c3a